bServer: true
serverIP: 192.168.1.116
port: 22822
autoIP: false
IP: 192.168.1.116
useUDP: false
sendTimes: 1
